interview-questions.md

```diff
@@ -8,36 +8,74 @@ Then, research the question to expand on your answer. Even if you feel you have
 
 1. As a developer, I am creating a Rails application with a model called Cohort and a model called Student. The cohort model has_many students, but OOPS! I forgot to add the foreign key. How can I fix this mistake? What is the name of the foreign key? Would the foreign key be on the Cohort model or the Student model?
 
-Your answer:
-
-Researched answer:
+Your answer: I am not sure I would have to reference my notes
+ 
+Researched answer: Student would be the froeign key and we would have to create syntax that identifies the Student key with the cohort primary key. The syntax would look like the following: class Student < ApplicationRecord
+  belongs_to :cohort
```

Comment:
You are thinking of associations here, but the question is assuming that the correct associations have been made, but the foreign key was not added when the Student model was generated. The question is asking how can you add a foreign key to a model that already exists?